libs/transloco-schematics/src/migrate/ngx-translate-migration.ts

```diff
@@ -158,14 +161,21 @@ export function run(path) {
   async function migrate(matchersArr, filesType) {
     console.log(`\nMigrating ${filesType} files 📜`);
     let spinner;
+    const isWindows = process.platform === 'win32';
+
     for (let i = 0; i < matchersArr.length; i++) {
       const { step, matchers } = matchersArr[i];
       const msg = `Step ${i + 1}/${matchersArr.length}: Migrating ${step}`;
       spinner = ora().start(msg);
       const noFilesFound = [];
       for (const matcher of matchers) {
         try {
-          await replaceInFile(matcher);
+          await replaceInFile({
+            ...matcher,
+            glob: {
+              windowsPathsNoEscape: isWindows,
```

Comment:
That's the main change. Prettier added a lot of noise to the PR.

Reply:
I wonder why 🤔 everything should be formatted.